src/assets/custom-bootstrap.scss

```diff
@@ -162,6 +162,10 @@ hr {
 
 @include media-breakpoint-up(sm) {
 
+  .modal-dialog {
+    max-width: 800px;
```

Comment:
I'm not sure this was ever updated to fix the "full width modals on medium-size screens" issue

Reply:
Will address this in master